src/ReactiveDomain.Testing/Specifications/MockRepositoryTests.cs

```diff
@@ -70,6 +70,13 @@ public void can_clear_queues()
                 .RepositoryEvents
                 .AssertEmpty();
         }
+        [Fact]
+        public void create_mock_repository_without_a_prefix()
+        {
+            var baseMockRepository = new MockRepositorySpecification();
+            var mockRepositoryWithoutPrefix = new MockRepositorySpecification(dataStore: baseMockRepository.StreamStoreConnection);
+            Assert.Equal("", mockRepositoryWithoutPrefix.Schema);
```

Comment:
What we should really test here is that when we get a stream name from that mock repo, it has no prefix. The test as it is now isn't actually testing the changed code.

Along the same lines, a test that we can get a correct stream name _with_ a prefix would also be good.